### PR TITLE
[homekit] Support UP/DOWN for WindowCoverings

### LINKED ISSUE #15561: [homekit] support UP/DOWN for blinds (instead of setPosition)
home app sends only setPosition, e.g. setPosition(70%). however, many blinds only support UP/DOWN/STOP. 
currently, these blinds would require a rule to convert setPosition to UP/DOWN + timer+ STOP
as it is pretty common requirement, it would be great to add this logic to homekit addon (similar to STOP simulation)

see https://community.openhab.org/t/homekit-and-somfy-roller-shutters/

### PR DESCRIPTION
**Pull request by ccutrer**
Fixes #15561

### CODE REVIEW

**Review by yfre — APPROVED** [review]
another interesting feature and pretty simple to implement
thank you Cody
LGTM

File: bundles/org.openhab.io.homekit/README.md [hunk 1]
```diff
@@ -466,6 +466,14 @@ Some blinds devices do support "STOP" command but would stop if they receive UP/
 Rollershutter window_covering "Window Rollershutter" {homekit = "WindowCovering"  [stop=true, stopSameDirection=true]}
  ```
 
+Some blinds devices do not support going to a specific position, even though they are modeled as Rollershutter items.
+The Home App only sends exact percentages by default.
+To avoid creating a rule, you can have the HomeKit addon translate 0%/100% to UP/DOWN commands.
+
+```java
+Rollshutter window_covering "Window Rollershutter" {homekit = "WindowCovering" [sendUpDownForExtents=true]}
```
Comment (maniac103) [inline bundles/org.openhab.io.homekit/README.md:474]: `Rollershutter`

**Comment by lsiepel** [comment]
Double check question. Does this feauture take the inverted flag into account? I remember i had multiple issues with different company's consider open/close (and so 0/100%) inverted to Apple.

**Comment by ccutrer** [comment]
> Double check question. Does this feauture take the inverted flag into account? I remember i had multiple issues with different company's consider open/close (and so 0/100%) inverted to Apple.

Yes. I had to double check because AbstractHomekitPositionAccessoryImpl has closedPosition/openPosition fields that hold 0/100 as appropriate depending on invertedness, and I went "oooh, wait, I'm using a literal. is this alright?". But yes, the conversion from HomeKit's value according to user configuration to the openHAB schema of 100=closed happens at the top of this method, so at this point 0 really means open and 100 really means closed.

**Pull request merged**